blueprints/instance-initializer-test/files/tests/unit/instance-initializers/__name__-test.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+import { initialize } from '<%= dependencyDepth %>/instance-initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+
+var appInstance;
+
+module('<%= friendlyTestName %>', {
+  beforeEach: function() {
+    Ember.run(function() {
+      var application = Ember.Application.create();
+      application.visit('/').then(function(instance) {
```

Comment:
Lets swap this out for `appliction.buildInstance()` (which is available from Ember 1.12.0 forward).  It is labeled as `@private`, but there is no other way to generate an `ApplicationInstance` instance (since `ApplicationInstance` isn't exported on the global).